### DISCUSSION TITLE
Understanding User management

### DISCUSSION THREAD

**Discussion by nodje** [dani-garcia/vaultwarden#2468] [General]
I'm not sure I understand what exists in terms of user management in Vaultwarden.

As there doesn't seem to be a superuser role, I'd expect to see, as the initial user created, a place where one can create, list and manage all existing users of the instance.

Is it something available? Or how should one go about this?

AFAIK:

To create a user, if SIGNUPS_ALLOWED=false, one has to go through an Organization and invite a user, which in turn, will be able to Create User from the main web page (that is IF INVITATIONS_ALLOWED=true, I'm not sure how this happens when INVITATIONS_ALLOWED=false)

To delete a user, the User itself has to login and delete the account in settings

**Comment by BlackDex** [comment]
Checkout https://github.com/dani-garcia/vaultwarden/wiki/Enabling-admin-page

**Comment by nodje** [comment]
Is there any settings I should be looking for specifically? 
Cause I haven't seen anything related to user management in admin settings

**Comment by BlackDex** [comment]
Just go to `/admin/users/overview` there is a link at the top called `Users`.

**Comment by nodje** [comment]
Many thanks for taking the effort to reply this dumb question.
As a matter of fact, I totally overlooked the white on black top level menu bar. 
I'm using a dark theme in my browser, which doesn't help to make the top noticable. Still I was totally focused on the white background main content.
Appologies
